web_poet/fields.py

```diff
@@ -0,0 +1,63 @@
+"""
+Module with helpers for defining Page Objects. It allows to define Page Objects
+in the following way:
+
+.. code-block: python
+
+    from web_poet import ItemPage, field, item_from_fields
+
+
+    class MyPage(ItemWebPage):
+        @field
+        def name(self):
+            return self.response.css(".name").get()
+
+        @field
+        def price(self):
+            return self.response.css(".price").get()
+
+        @field
+        def currency(self):
+            return "USD"
+
+        async def to_item(self):
+            return await item_from_fields(self)
```

Comment:
It could also be 

```py
class MyPage(ItemPage[Product]):
    # ...
```

This should allow to set proper typing info for the `to_item` method, and provide a default implementation.

Reply:
We should also think about interaction with handle_urls decorator: in the example below it seems there is no need for `overrides` keyword argument:

```py
@handle_url("foo.com")
class MyPage(ItemPage[Product]):
    # ...
```